src/main/java/net/openhft/chronicle/wire/converter/SymbolsLongConverter.java

```diff
@@ -149,6 +154,9 @@ public void append(Bytes<?> text, long value) {
             text.append(decode[v]);
         }
 
+        if (!text.isEmpty())
+            text.unwrite(text.writePosition() - 1, 1);
+
```

Comment:
Actually this isn't backward or forward compatible. I really like it though, perhaps we make it a separate implementation and deprecate this one....

e.g. when encoded by the new impl and read by the old impl you get
`5HELLO`

and when encoded by the old impl and read by the new impl you get:
`ELLO`